tetris-game/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
I think we need to remove this file from future PR's so that we only have one README, approved for now though